main.py

```diff
@@ -148,8 +149,7 @@ def update(self):
 	def render(self):
 		self.clear()
 
-		world_batch = world.draw(self, self.sprites)
```

Comment:
This was crazy expensive. 1GB of memory per second. Garbage collector made it glitch when dumping.